README.md

```diff
@@ -9,4 +9,4 @@
 > npm start
 
 ### Run this command to build your webpack
-> npm run build
\ No newline at end of file
+> npm run build# todo-list
```

Comment:
- [ ] Please, consider modifying your README file, using the provided [MISCOVERSE template](https://github.com/microverseinc/readme-template/blob/master/README.md), as it is a general requirement, which will make your repo professional 💯 🥇